src/app/component/SliderWidget/SliderWidget.component.js

```diff
@@ -58,6 +105,7 @@ export default class SliderWidget extends PureComponent {
               block="SliderWidget"
               elem="Figure"
               key={ i }
+              style={ { width: `${this.state.slideWidth}%` } }
```

Comment:
Setting style this way will prevent Slider component in itself from being used to display multiple simultaneous slides easily.
I suggest instead create a ref map with children in Slider component and calculate the slide width in px there and set it that way.

Reply:
That is correct + I would recommend using CSS variables, not direct property setting.